### ISSUE TITLE
[TF2] Quickfix Uber prevents Detonator jumps.

### ISSUE THREAD

**Issue by pikasalt** [ValveSoftware/Source-1-Games#4842]
When under the effects of a Quickfix Uber, a Pyro with the detonator is unable to perform blast jumps.

**Comment by Mishih** [comment]
Yes, they should fix it. Maybe this is due to the fact that if uber is activated, then you have a resistance to repulsion.
